deploy/deployment.yaml

```diff
@@ -17,5 +17,22 @@ spec:
       serviceAccountName: pipelines-app-delivery
       containers:
         - name: pipelines-app-delivery-http
-          image: quay.io/redhat-developer/gitops-backend:v0.0.2
+          image: quay.io/redhat-developer/gitops-backend:v0.0.3
           imagePullPolicy: Always
+          env:
+          - name: INSECURE
+            value: "true"
+          volumeMounts:
+            - mountPath: "/etc/gitops/ssl"
+              name: backend-ssl
+              readOnly: true
+          ports:
+          -  containerPort: 8080
+          volumes:
```

Comment:
There's already a `volumes` field in `spec`. I guess this is repeated inside `containers`